#getting draw information of a stored object
#objectID		
uint32 objectID
---
#object shape 1 rectangle 2 ellipse
uint32 shape
#object width
float32 width
#object height
float32 height
#object color in hex eg red FF0000 green is 00FF00 and blue is 0000FF
string color
